refresh/index.js

```diff
@@ -300,9 +299,12 @@ module.exports = Generator.extend({
       this.itemsToIgnore = []
 
       // Package dependencies to add to Package.swift
-      // eg this.dependencies.push('.Package(url: "https://github.com/IBM-Swift/Kitura.git", majorVersion: 1, minor: 7),')
+      // eg this.dependencies.push('.package(url: "https://github.com/IBM-Swift/Kitura.git", .upToNextMinor(from : "1.7.0")),')
```

Comment:
Ooh, copy these two lines (the deleted comment and the new comment) up to the other file and you have your example input/output! :)